src/acceptance/administrator/components/com_menu/MenuItemCest.php

```diff
@@ -0,0 +1,197 @@
+        $I->checkForPhpNoticesOrWarnings();
+        $I->click(MenuItemListPage::$selectMenu);
+        $option = $I->grabTextFrom(MenuItemListPage::$selectMainMenu);
+        $I->selectOption(MenuItemListPage::$selectMenu, $option);
+        // Search For Menu Item
+        $I->searchForItem($this->menuItemName);
+        $I->click(MenuItemListPage::$checkItemOne);
+        // Unpublish
+        $I->clickToolbarButton('unpublish');
+        // Assertion
+        $I->setFilter('select status', 'Unpublished');
+        $I->searchForItem($this->menuItemName);
+        $I->seeElement(['link' => $this->menuItemName]);
+    }
+
+    /**
+     * Publish a menu
+     *
+     * @param \AcceptanceTester $I The AcceptanceTester Object
+     *
+     * @since 4.0.0
+     *
+     * @return void
+     */
+    public function publishMenuItem(\AcceptanceTester $I)
+    {
+        $I->comment('I am going to publish a menu');
+        $I->doAdministratorLogin();
+        // URL
+        $I->amOnPage(MenuItemListPage::$url);
+        // Search For Menu Item
+        $I->searchForItem($this->menuItemName);
+        $I->click(MenuItemListPage::$checkItemOne);
+        // Publish
+        $I->clickToolbarButton('publish');
+        // Assertion
+        $I->setFilter('select status', 'Published');
+        $I->searchForItem($this->menuItemName);
+        $I->seeElement(['link' => $this->menuItemName]);
+    }
+
+    /**
+     * Check In A Menu Item
+     *
+     * @param \AcceptanceTester $I The AcceptanceTester Object
+     *
+     * @since 4.0.0
+     *
+     * @return void
+     */
+    public function checkInMenuItem(\AcceptanceTester $I)
+    {
+        $I->comment('I am going to check in a menu item');
+        $I->doAdministratorLogin();
+        // URL
+        $I->amOnPage(MenuItemListPage::$url);
+        $I->checkForPhpNoticesOrWarnings();
+        // Search For Menu Item
+        $I->searchForItem($this->menuItemName);
+        $I->click(MenuItemListPage::$checkItemOne);
+        // check in button
+        $I->clickToolbarButton('check-in');
+    }
+
+    /**
+     * Set A Menu Item To Home
+     *
+     * @param \AcceptanceTester $I The AcceptanceTester Object
+     *
+     * @since 4.0.0
+     *
+     * @return void
+     */
+    public function setHomeMenuItem(\AcceptanceTester $I)
+    {
+        $I->comment('I am going to set a menu item to home');
+        $I->doAdministratorLogin();
+        // URL
+        $I->amOnPage(MenuItemListPage::$url);
+        $I->checkForPhpNoticesOrWarnings();
+        $I->click(MenuItemListPage::$selectMenu);
+        $option = $I->grabTextFrom(MenuItemListPage::$selectMainMenu);
+        $I->selectOption(MenuItemListPage::$selectMenu, $option);
+        // Search For Menu Item
+        $I->searchForItem($this->menuItemName);
+        $I->click(MenuItemListPage::$checkItemOne);
+        //Home button doesn't exist in JoomlaBrowser.php file (joomla-browser)
+        $I->click(MenuItemListPage::$homeButton);
```

Comment:
where are the assertions?, asserting of the Menu item is available, asserting the actions are behaving as expected

Reply:
There is no option in filter to check for 'Home' and 'check-in'.
Should I assert 'home'  using  system-container Successful message?